Assets/Scripts/Destructible.cs

```diff
@@ -0,0 +1,82 @@
+﻿using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+/***********************************************************
+ * Script will change an object's texture to appear damage
+ * Accomplished by changing Sprites when object is within a
+ * certain range (Health calculated in percent)
+ ***********************************************************/
+public class Destructible : Entity
+{
+    public Sprite[] Sprites; //The different textures go into this array
+    private float[] HealthPercents;
+    private float MaxHealth; //Constant value; an objects maximum health
+    private float prevHealth;
+
+    //Private method determines the cutoffs on percent the ranges based on the number of Sprites
+    private float[] CreatePercents(int numSprites)
+    {
+        float[] HealthPercents = new float[numSprites];
+
+        HealthPercents[0] = 100;
+        if (numSprites > 1)
+        {
+            float DISTANCE = 100 / (numSprites - 1); //Constant value; Distance from one cutoff to another
```

Comment:
fyi, usually all caps variables are saved for preprocessor macros. also, C# variable naming convention is camelCase with the first character lowercase. This isn't something we care about, just wanted to let you know